### DISCUSSION TITLE
how to enable rust-analyzer to show *more than just type* in inlay hint (grey colored text).

### DISCUSSION THREAD

**Discussion by Ashu999** [rust-lang/rust-analyzer#16857] [Q&A] [answered]
I want to see not just "String" but also "<String as Add<&String>>::Output". (as shown in line 3 of attached image, this screenshot is from a video I saw online)

![types and function](https://github.com/rust-lang/rust-analyzer/assets/28844854/f789688d-fc2b-4277-80eb-af974618888b)

**Comment by lnicola** [comment, marked as answer]
I think that's what we show when we don't know what the type is, so it's a bug, not a feature to enable.
